helix-core/src/main/java/org/apache/helix/ConfigAccessor.java

```diff
@@ -44,6 +44,7 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+
```

Comment:
Thanks for doing this! 

1. Does this include the new endpoints?
2. LOG is the convention we use throughout our codebase. Could you also quickly change logger's to LOG?

Reply:
For your questions:
1. The new endpoints will be changed in the other PR. This one will be merged before that one.
2. Updated to LOG.